README.md

```diff
@@ -69,29 +67,29 @@ Only the `source` import source phase is specified by this proposal.
 
 ### Defining Module Source
 
-Module source is defined through the [ECMA-262 ES modules HostLoadImportedModule refactoring][],
-which moves the construction of module records to the host, and on which we can define a
-`[[ModuleSourceObject]]` custom object to be returned by module source phase that is compatible
-with module expressions.
+The object provided by the module source phase should have a prototype set to the
+`%AbtractModuleSourcePrototype%` object, defined by this specification to be a minimal
+shared base prototype for a compiled modular resource.
+
+In addition we define the `sourceType` getter returning the type string
+corresponding to the name of the specific module source subclass.
```

Comment:
I believe we’re converging on omitting this in favor of an `Symbol.toStringTag` getter on the `%AbstractModuleSourcePrototype%` that is sufficient to distinguish the module source brand and also differentiate implementations by consulting internal slots. A `toStringTag` of `"ModuleSource"` would imply source text for this language, as opposed to `"WebAssembly.Module"`.

For purposes of consensus, I am not picky about these particulars. None of the motivating use cases I am tracking require a brand check or module source type reflection. Module source objects can be opaque tokens for import virtualization and only need to reveal their language-agnostic `bindings` for loader virtualization, bundling, or import graph analysis.

Reply:
Thanks, this was also missed in the refactor.